recipes-dev/pycentroids/meta.yaml

```diff
@@ -15,7 +15,9 @@ requirements:
     - setuptools
     - python
     - numpy x.x
+    - scipy
```

Comment:
As this recipe was change, we will have to increment the build number above, otherwise the package won't be built.

Reply:
By the way, why it was set to 10? We usually put `0` as a default. Conda forge may have `1000` to hint which compiler to use, iirc. Right, @CJ-Wright?

Reply:
@mrakitin I don't think we need to bump the build number because I bumped the version number?

Reply:
I can reset to zero when I do the next version bump, I entered 10 in testing

Reply:
This is in the dev version which we are not auto-building and should not be manually bumped ;)

Reply:
So this is a mute point?

Reply:
Ah, right, didn't pay attention it was a dev recipe. Probably no point to update it as we don't support it *right now*, but may do in the future.